contracts/PBAB+Collabs/GenArt721MinterBurner_PBAB.sol

```diff
@@ -46,6 +46,9 @@ contract GenArt721MinterBurner_PBAB is ReentrancyGuard {
 
     /// ERC20 address => burn during purchase
     mapping(address => bool) public burnERC20DuringPurchase;
+    // don't rely on optional ERC20 burn() to burn tokens, send here instead
+    address internal constant ERC20_BURN_ADDRESS =
+        0x000000000000000000000000000000000000dEaD;
```

Comment:
Just curious use of `0x000000000000000000000000000000000000dEaD` vs `address(0)` here. I have no strong opinion whatsoever aside from slight preference towards standardizing our approach here in our contracts repo (that is, we should maybe make an opinion and add it to the README or a new STYLE.md file).

Reply:
Ah whoops, I just saw the rationale for this above in https://github.com/ArtBlocks/artblocks-contracts/pull/197#issuecomment-1165990018

This approach SGTM and is probably just something worth documenting more explicitly outside the file, perhaps in starting a new STYLE.md. 

In any case, that can be handled in a new separate PR ofc.